src/components/organisms/tables/OrdersTable/OrdersTable.tsx

```diff
@@ -119,6 +123,33 @@ const OrdersTable: FC = () => {
     },
   })
 
+  const handleModifiedFilterChange = useCallback(
+    (filters?: FilterFunctionType) => {
+      let currentFilters = filters
```

Comment:
**[non-blocker]**

Not too important, but would suggest avoiding any data mutations, when dealing with React projects. They can have some unintended side effects. This is mostly related to anything stored to state and/or used by react for rerendering. It is fine to use in some cases, but usually we try to avoid doing it in React projects, to make sure we don't end up with some unintended behaviour + to reduce the chance that people will use that code as example in some other context, where it will cause issues.

**Note**: It doesn't cause any issues here

Reply:
Noted! I thought the code would be easier to read if I didn't call the `if (handleFilterChange) { handleFilterChange(...) }` twice with the two possible options separately.

Reply:
There is another option to do sth like 

  const currentFilters = {
    ...rest,
    ...(isEmpty(modifiedLanguageDirections)
      ? { language_directions: modifiedLanguageDirections }
      : {}),
  }
  
  But yeah, not really important, can stay the way it is